src/Security/Authentication/test/FacebookTests.cs

```diff
@@ -118,7 +118,7 @@ public async Task ChallengeWillIncludeScopeAsConfigured()
             app => app.UseAuthentication(),
             services =>
             {
-                services.AddAuthentication().AddFacebook(o =>
+                services.AddAuthentication(o => o.DisableAutoDefaultScheme = true).AddFacebook(o =>
```

Comment:
These tests are only testing very specific paths, setting the default scheme causes some of the other schemes to now have values which causes exceptions in these tests which are unrelated to what they are trying to test, easiest to just disable the new behavior